gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/service/NackAckPreparationServiceTest.java

```diff
@@ -67,35 +63,28 @@ class NackAckPreparationServiceTest {
     @Captor
     private ArgumentCaptor<MigrationStatus> migrationStatusCaptor;
 
-    @Test
-    public void When_SendNackMessageRCMR_WithNoErrors_Expect_ShouldUpdateLog() throws JAXBException {
-        RCMRIN030000UKMessage payload = unmarshallString(
-                readInboundMessagePayloadFromFile(), RCMRIN030000UKMessage.class);
-
-        when(sendNACKMessageHandler.prepareAndSendMessage(any(NACKMessageData.class))).thenReturn(true);
-
-        assertTrue(nackAckPreparationService.sendNackMessage(LARGE_MESSAGE_GENERAL_FAILURE, payload, CONVERSATION_ID));
-        verify(migrationStatusLogService)
-            .addMigrationStatusLog(ERROR_LRG_MSG_GENERAL_FAILURE, CONVERSATION_ID, null, LARGE_MESSAGE_GENERAL_FAILURE.getCode());
-    }
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void When_SendNackMessageRCMR_Expect_ShouldUpdateLogAndReturnPrepareAndSendMessageResult(boolean expectedResult)
```

Comment:
expectedResult -> ableToSendMessage ?

Reply:
have renamed this variable as suggested, and `actualResult` to `hasMessageBeenSent`